src/test/java/upbrella/be/rent/controller/RentControllerTest.java

```diff
@@ -109,4 +115,164 @@ void returnUmbrellaTest() throws Exception {
                                         .description("데이터 값이 없습니다.")
                         )));
     }
+
+    @DisplayName("사용자는 우산 대여 내역을 조회 할 수 있다.")
+    @Test
+    void showAllRentalHistoriesTest() throws Exception {
+
+        RentalHistoriesPageResponse response = RentalHistoriesPageResponse.builder()
+                .rentalHistoryResponsePage(List.of(RentalHistoryResponse.builder()
+                        .id(1L)
+                        .name("사용자")
+                        .phoneNumber("010-1234-5678")
+                        .rentStoreName("대여점 이름")
+                        .rentAt(LocalDateTime.of(2023, 7, 18, 0, 0, 0))
+                        .elapsedDay(3)
+                        .umbrellaId(30)
+                        .returnStoreName("반납점 이름")
+                        .returnAt(LocalDateTime.of(2023, 7, 23, 0, 0, 0))
+                        .totalRentalDay(5)
+                        .refundCompleted(true)
+                        .etc("기타")
+                        .build())
+                ).build();
+
+        mockMvc.perform(
+                        get("/rent/histories")
+                )
+                .andDo(print())
+                .andExpect(status().isOk())
+                .andDo(document("show-all-rental-histories-doc",
+                        preprocessResponse(prettyPrint()),
```

Comment:
```suggestion
                        getDocumentRequest(),
                        getDocumentResponse(),
```
RestDocs 템플릿을 추가해서 pull한번 받고 이렇게 변경해주세요.
자세한 내용은 (#30) 참고바랍니다.

Reply:
나머지 API도 동일하게 변경하면됩니다